src/sync/update_tx.js

```diff
@@ -37,6 +38,22 @@ async function updatePendingTxs(db) {
 
 // Update the Transaction info
 async function updateTx(tx) {
+  // sendtoaddress does not use the same confirmation method as EVM txs
+  if (tx.type === 'TRANSFER' && tx.token === 'QTUM') {
+    const txInfo = await wallet.getTransaction({ txid: tx.txid });
+    const blockInfo = await blockchain.getBlock({ blockHash: txInfo.blockhash });
```

Comment:
The request to get current block should be called only once per sync. If needed, we can pass it as a parameter in `updateTx(tx, currenBlock, currentBlockTime)`

Reply:
fixed